src/octopeer-github/main/Status.ts

```diff
@@ -81,25 +90,31 @@ class Status {
      * Or it sets the OFF flag if the logging is not enabled.
      * @param status
      */
-    // TODO: Will be fixed in #44, due to bug in Settings.
     public set(status: StatusCode) {
-        // if (Settings.getLogging()) {
+        if (Options.getLogging()) {
             this.setter(status);
-        // } else {
-        //     this.setter(StatusCode.OFF);
-        // }
+        } else {
+            this.setter(StatusCode.OFF);
+        }
     }
 
     /**
      * Helper method for set() in order to prevent duplicate code.
      * @param status
      */
-    private setter(status: StatusCode) {
+    public setter(status: StatusCode) {
```

Comment:
Yes, it used to be, but then we swapped namespaces and classes. Will fix in #44 ( #70 )